### PR TITLE
Fix backup k8s job

### PR DESCRIPTION
**Pull request by hogo6002**
1. Add log config for run_cron jobs
2. Import data_types from clusterfuzz internal package to update ndb kind_map

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]

File: src/clusterfuzz/_internal/bot/tasks/cron/backup.py [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 from clusterfuzz._internal.base import utils
 from clusterfuzz._internal.config import local_config
+# pylint: disable=unused-import
+# The ndb module updates _kind_map from imported classes
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/bot/tasks/cron/backup.py:28]: nit: reword this to "This is required to populate the ndb _kind_map".

**Review by oliverchang — APPROVED** [review]
LGTM with one nit.
